src/platforms/dotnet/guides/xamarin/troubleshooting.mdx

```diff
@@ -65,6 +65,21 @@ Using Reference will not work, it must be PackageReference.
 
 More information about this problem can be found here: https://github.com/mono/mono/issues/20805
 
+## Android 
+
+### Event/Transaction not sent
+
+You may encounter the following issue when sending Events or Envelopes:
+
+`The SSL connection could not be established, CERTIFICATE_VERIFY_FAILED`
+
+A quick solution is to alter your DSN with an alternative one, for that, remove the values between `@` and `sentry.io` of your DSN.
```

Comment:
We want to avoid language like "quick" or "simple" in case the process turns out to not be quick or simple for the user, for whatever reason.

```suggestion
One solution is to alter your DSN; to do this, remove the values between `@` and `sentry.io` in your DSN.
```

Reply:
Fair point 😄